Basic-Video-Chat-ConnectionService_FCM/README.md

```diff
@@ -0,0 +1,62 @@
+# Basic Video Chat ConnectionService FCM Java
+
+This application provides a completed version of the OpenTok [Basic Video Chat tutorial](https://tokbox.com/developer/tutorials/android/) for Android (differing only in some additional validation checks). Upon deploying this sample application, you should be able to have two-way audio and video communication using OpenTok.
+
+Main features:
+* Connect to an OpenTok session
+* Publish an audio-video stream to the session
+* Subscribe to another client's audio-video stream
```

Comment:
Shouldn't we improve this README? I was expecting to find an introduction about the connection services and FCM.